GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+# Fn Project Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Fn Project community. Common terminology used in this governance document are listed below:
+  * Team members: members of the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * Maintainers: Maintainers lead an individual project or parts thereof.
+  * Projects: A single repository in the [Fn Project GitHub organization](https://github.com/fnproject).
+  * The Fn Project: The sum of all activities performed under this governance, concerning one or more repositories or the community. 
+
+## Values
+The Fn Project exists to serve the CNCF ecosystem. Thus developers and community are expected to follow the values defined in the [CNCF Charter](https://www.cncf.io/about/charter/), including the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). Furthermore, the Fn Project community strives for kindness, giving feedback effectively, and building a welcoming environment. The Fn Project developers generally decide by consensus and only resort to conflict resolution by a majority vote if consensus cannot be reached.
+
+## Projects
+Each project must have a `MAINTAINERS.md` file with at least one maintainer. Where a project has a release process, access and documentation should be such that more than one person can perform a release. Releases should be announced in the [Release category in forums.fnproject.io](https://forums.fnproject.io/c/release) . Any new projects should be first proposed in the [Fn Project forums](https://forums.fnproject.io/) following the voting procedures listed below. When a project is no longer relevant it should be moved to the fnproject-junkyard GitHub organization.
```

Comment:
might we say 'major' or 'major/minor' release? slight technicality that every commit is a release at the moment (which is ok). don't mean to open the release can of worms here, necessarily

Reply:
Added 'Major'.